tests/old_suite/runtests.py

```diff
@@ -107,15 +107,12 @@ def __init__(self):
             'basic/test_pkg_structures': ['pkg_resources'],
 
             'libraries/test_gst': ['gst'],
-            'libraries/test_twisted_qt4reactor': ['twisted', 'PyQt4', 'qt4reactor'],
```

Comment:
Remove the changes to this file; they aren't needed - the old suite is never run by us.

Reply:
will revert all changes to tests/old_suite/. in next push

Reply:
will remove all changes to tests/old_suite in next push

Reply:
Please, please also remove these Qt4 tests from the old suite! The old-suite is a pain anayway and as smaller as better.

Reply:
so .. should I remove the old_suite/ files? I think it is worth removing them to have no references to pyqt4 in the codebase; @Legorooj  ?

Reply:
Please remove the Qt4-related stuff from the old suite, as you did first. Thanks and sorry for the confusion.